test/duke/command/UserCommandsTest.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class UserCommandsTest {
+
+    @Test
+    void testGetTodoCommandWord() {
+        assertEquals("todo", UserCommands.TODO.getCommandWord());
+    }
+
+    @Test
```

Comment:
Also would be good to add a javadoc on what the test is supposed to be about to make it clearer to reviewers what is the expected behaviour for the Unittest